src/NServiceBus.Persistence.DynamoDB/SynchronizedStorage/DynamoDBSynchronizedStorageSession.cs

```diff
@@ -11,7 +11,7 @@
 
     class DynamoDBSynchronizedStorageSession : ICompletableSynchronizedStorageSession, IDynamoDBStorageSession
     {
-        internal StorageSession storageSession;
+        internal StorageSession storageSession = null!;
```

Comment:
the whole session concept kinda expects that access to the session is only valid once it has been opened so I decided to not make this nullable. Happy to discuss though.

Reply:
Nah makes sense